src/api/canvas-props.ts

```diff
@@ -2,8 +2,10 @@ import { RenderProps } from "@react-three/fiber";
 import { PropsWithChildren } from "react";
 
 export interface CanvasProps extends Omit<RenderProps<HTMLCanvasElement>, 'frameloop'>, PropsWithChildren {
+  id?: string;
```

Comment:
You think in here ([link](https://github.com/RodrigoHamuy/react-three-map/blob/main/src/core/use-canvas.ts#L7)) we can do the following, please?

```ts
export interface useCanvasProps extends Omit<CanvasProps, 'id'|'beforeId'> {
```